### PR TITLE
amp-addthis: Update availability documentation since it is live

### PR DESCRIPTION
**Pull request by aghassemi**

### CODE REVIEW

File: extensions/amp-addthis/amp-addthis.md [hunk 1]
```diff
@@ -117,4 +113,4 @@ AddThis Data may consist of the following:
 AddThis Data will be treated as personal information to the extent required under applicable law.
 Publishers are required per the AddThis Terms of Service to obtain all necessary End User consents
 and authorizations, and provide any required notices for the provision of AddThis Data collected
-from End Users to Oracle.
\ No newline at end of file
+from End Users to Oracle.
```
Comment (cathyxz) [inline extensions/amp-addthis/amp-addthis.md]: Is this removal intentional? If so, nit: missing full stop.

Comment (aghassemi) [reply extensions/amp-addthis/amp-addthis.md]: The line is not being removed, the diff is misleading. Just a whitespace removal that my IDE does by default. (a newline is being removed)
